tests/conftest.py

```diff
@@ -291,17 +285,28 @@ def sklearn_full_version(request):
     return request.config.getoption("--sklearn-full-version")
 
 
-@pytest.fixture(scope="module", params=[TensorFlow._LATEST_1X_VERSION, LATEST_VERSION])
+@pytest.fixture(scope="module")
+def tf_latest_version(request, tf_full_version):
+    return request.param
```

Comment:
I think you can scrap this in favor of `tf_full_version`?

Reply:
done